CredentialProvider.Microsoft/CredentialProviders/Vsts/IAuthUtil.cs

```diff
@@ -81,7 +84,7 @@ public async Task<Uri> GetAadAuthorityUriAsync(Uri uri, CancellationToken cancel
             var aadBase = UsePpeAadUrl(uri) ? "https://login.windows-ppe.net" : "https://login.microsoftonline.com";
             logger.Verbose(string.Format(Resources.AADAuthorityNotFound, aadBase));
 
-            var tenant = EnvUtil.MsalEnabled() ? OrganizationsTenant: CommonTenant;
+            var tenant = EnvUtil.MsalEnabled() ? MsaTenant.ToString() : CommonTenant;
```

Comment:
Is it safe to assume this is MSA?

Reply:
I believe that's what the "organizations" was doing before, but will confirm

Reply:
'organizations' includes work and school accounts, and MSA users if passthrough is enabled. If it was a work or school account then there would have been a tenant id available from the unauthenticated request against the Azure DevOps endpoint, which leaves only MSA passthrough users.